backend/controllers/userController.js

```diff
@@ -0,0 +1,36 @@
+import generatePassword from '../utils/generatePassword';
+import userService from '../services/userService';
+import generateToken from '../utils/generateToken';
+
+
+const _ = require('lodash');
+
+
+const createUser = async (req, res) => {
```

Comment:
Please let's stick to classes

Reply:
I have changed that you could kindly review it again